src/formatHit.js

```diff
@@ -83,7 +121,8 @@ export default function formatHit({
         lat: hit._geoloc.lat,
         lng: hit._geoloc.lng,
       },
-      postcode: hit.postcode && hit.postcode[0],
+      postcode,
+      postcodes: hit.postcode ? hit.postcode : undefined,
```

Comment:
Shouldn't it be `postcodes: hit.postcodes ? hit.postcodes : undefined,`?

Reply:
The raw response field for the postcode array is `postcode`, so it's not a typo!

Reply:
Oh nice catch 👍